dali/test/python/test_operator_readers_webdataset_requirements.py

```diff
@@ -246,6 +246,46 @@ def test_sharding():
             math.ceil(num_samples / num_shards / test_batch_size) * 2,
         )
 
+
+def test_pax_format():
+    global test_batch_size
+    num_samples = 1000
+    tar_file_path = os.path.join(get_dali_extra_path(), "db/webdataset/MNIST/devel-0.tar")
+    pax_tar_file_path = os.path.join(get_dali_extra_path(), "db/webdataset/pax/devel-0.tar")
+    index_file = generate_temp_index_file(tar_file_path)
+
+    extract_dir = generate_temp_extract(tar_file_path)
+    equivalent_files = sorted(
+        glob(extract_dir.name + "/*"), key=lambda s: int(s[s.rfind("/") + 1 : s.rfind(".")])
+    )
```

Comment:
This operation is not needed in this tests.

Reply:
Fixed